src/main/java/io/cosmosoftware/kite/interfaces/CommandMaker.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2018 Cosmo Software
```

Comment:
wrong copyright

Reply:
2019 ? I copied the file from kite server

Reply:
```
/*
 * Copyright (C) CoSMo Software Consulting Pte. Ltd. - All Rights Reserved
 */

```